apis/core/v1alpha2/serving_conversion.go

```diff
@@ -244,7 +244,7 @@ func (dst *Serving) convertServingFrom(src *v1beta1.Serving) error {
 		}
 	}
 
-	if src.Spec.Annotations != nil {
+	if src.Spec.Annotations != nil && src.Spec.Runtime == v1beta1.Async {
```

Comment:
Do we need to check runtime as well in this [line](https://github.com/OpenFunction/OpenFunction/blob/0b8fd9e7c688bd85df8bef7f277793fc3fa82e7c/apis/core/v1alpha2/function_conversion.go#L422)?

Reply:
I think no need, because the sync function has no inputs

Reply:
do we have logic to reject/ignore `inputs` for sync function?

Reply:
Looks like no, let me add this condition judgment.